dbt/models/churnado_update_features.sql

```diff
@@ -0,0 +1,23 @@
+select
+  u.id as user_id
+  , date(date_trunc('week', up.created_at)) as week
+  , datediff(week, dateadd(week, -8, '{{ var('t') }}'), date_trunc('week', up.created_at)) as week_number
+  , count(distinct up.id) as number_of_updates
+  , count(distinct date(up.created_at)) as number_of_days_with_updates
+  , count(distinct date_trunc('week', up.created_at)) as number_of_weeks_with_updates
+  , count(distinct case when up.client_type = 'Extension' then up.id else null end) as number_of_extension_updates
+  , count(distinct p.id) as number_of_profiles
+  , coalesce(count(distinct e.id), 0) as number_of_update_errors
+from dbt.users as u
+join dbt.profiles as p
+  on u.id = p.user_id
+join dbt.updates as up
+  on p.id = up.profile_id
+  -- only want updates created in Buffer (or do we?)
```

Comment:
I'd say so! We can also add that as an extra dimension and grab all of them.

Reply:
Awesome! Sounds good :)